### DISCUSSION TITLE
Compatible Themes

### DISCUSSION THREAD

**Discussion by C4E5G3** [nix-community/nixvim#1734] [Q&A] [answered]
Which themes work out of the box with Nixvim?

Is there a good plugin for constructing a bespoke colour scheme?

**Comment by MattSturgeon** [comment, marked as answer]
You can install any colorscheme packaged as a vim plugin, then use the `colorscheme` option to set it.

Nixvim also has module options for a fair amount of colorscheme plugins, see the "colorschemes" section in [the docs](https://nix-community.github.io/nixvim/).

To make a custom colorscheme, you might find the base16 plugin interesting (`colorschemes.base16`).
